docs/source/migration/4.0.mdx

```diff
@@ -201,4 +201,5 @@ The AST classes (`GQLNode` and subclasses) as well as `Introspection` classes ar
 
 ## Gradle configuration
 
-Publishing is no longer configured automatically.
+* Publishing is no longer configured automatically.
+* Because Apollo Kotlin now supports different operation manifest formats, `operationOutput.json` has moved from `"build/generated/oeprationOutput/apollo/$service/operationOutput.json"` to `"build/generated/manifest/apollo/$service/operationOutput.json"` 
```

Comment:
```suggestion
* Because Apollo Kotlin now supports different operation manifest formats, `operationOutput.json` has moved from `"build/generated/operationOutput/apollo/$service/operationOutput.json"` to `"build/generated/manifest/apollo/$service/operationOutput.json"` 
```